bpfctl/src/main.rs

```diff
@@ -105,7 +105,10 @@ async fn main() -> anyhow::Result<()> {
         .domain_name("localhost")
         .ca_certificate(ca_cert)
         .identity(identity);
-    let channel = Channel::from_static("http://[::1]:50051")
+    let endpoint = &config.grpc.endpoint;
+    // TODO: Use https (https://github.com/redhat-et/bpfd/issues/396)
+    let address = format!("http://{}:{}", endpoint.address, endpoint.port);
```

Comment:
Just realized this will probably not work with and IPv6 address, I'll change it.